akka-docs/src/main/paradox/stream/operators/Source/completionStage.md

```diff
@@ -30,9 +39,9 @@ For the corresponding operator for the Scala standard library `Future` see @ref:
 
 @@@
 
-## Example
 
-Java
-:  @@snip [SourceFromCompletionStage.java](/akka-docs/src/test/java/jdocs/stream/operators/source/FromCompletionStage.java) { #sourceFromCompletionStage }
+@@@div { .group-scala }
 
 For the corresponding operator for the Scala standard library `Future` see @ref:[future](future.md).
+
+@@@
```

Comment:
We don't need this block twice in the same page?

Reply:
I took https://doc.akka.io/docs/akka/current/stream/operators/Source/completionStage.html as the inspiration as it seemed the most complete of the 4 pages (`completionStage`, `completionStageSource`, `future`, `futureSource`) and followed along. Happy to remove the duplicate message.